test/e2e/util.go

```diff
@@ -2266,13 +2266,9 @@ func waitForNodeToBeNotReady(c *client.Client, name string, timeout time.Duratio
 
 func isNodeConditionSetAsExpected(node *api.Node, conditionType api.NodeConditionType, wantTrue bool) bool {
 	// Check the node readiness condition (logging all).
-	for i, cond := range node.Status.Conditions {
-		Logf("Node %s condition %d/%d: type: %v, status: %v, reason: %q, message: %q, last transition time: %v",
-			node.Name, i+1, len(node.Status.Conditions), cond.Type, cond.Status,
-			cond.Reason, cond.Message, cond.LastTransitionTime)
+	for _, cond := range node.Status.Conditions {
```

Comment:
It actually is - there are retries. But you're right, I probably can do it here as well.